### PR TITLE
Music Band Web Page: Final Version

### PR DESCRIPTION
**Pull request by Santiago220991**
# Tasks done:
- Added main page mobile version.
- Added about page mobile version.
- Added main page desktop version.
- Added about page desktop version.
- Added transitions and animations to the main page desktop version.
- Changed presentation video.

Link to loom video:
https://www.loom.com/share/5df4b8f3cf2341db8320ef4520072415

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -116,7 +117,7 @@ <h3>Shop</h3>
             
         </div>
         <div class="information-button">
-            <a href="">Purchase tickets</a>
+            <a href="">Purchase Tickets</a>
         </div>
       </section>
       <section class="band" id="band">
```
Comment (thecodechaser) [inline index.html:123]: - [ ] You did a very good job on the project but for this project, you are supposed to stick to the [Behance ](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) design as much as possible. When we hover the mouse over cards a white border around cards should appear as on the design, Please apply hover effects to your cards so the cards show a white border when we hover the mouse over them.

### See the image for reference 

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED 🔴🟡 🔴 

## Hi @Santiago220991👋 

### You are doing a marvelous job on the capstone project and this is one of the best projects I have seen so far!
I really like your presentation for the project 👍

There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- Good commit messages. ✔️
- PR has a good description and title ✔️ 



## Required Changes ♻️
- [ ]  In the presentation video you should show some piece of code that you are very proud of, Please make sure your presentation video includes that.
- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @Santiago220991 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## OPTIONAL

- [ ]  Your video presentation is wonderful 🥇. Please send the video link to the Readme file.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Santiago220991 — COMMENTED** [review]
Testing

**Pull request merged**